src/Evolution/Systems/Cce/Actions/InsertInterpolationScriData.hpp

```diff
@@ -60,6 +60,43 @@ struct InsertIntoInterpolationManagerImpl<::Tags::Multiplies<LhsTag, RhsTag>> {
                                        get(rhs_data).data());
   }
 };
+
+template <typename Tag, typename ParallelComponent, typename DbTags,
+          typename Metavariables>
+void output_impl(const db::DataBox<DbTags>& box,
+                 Parallel::GlobalCache<Metavariables>& cache) noexcept {
+  const size_t observation_l_max = db::get<Tags::ObservationLMax>(box);
+  const size_t l_max = db::get<Tags::LMax>(box);
```

Comment:
I have found it a lot clearer to pass explicit arguments to functions instead of a `DataBox`, whenever possible. I think it's generally very helpful to separate the code layer that deals with the DataBox and tags from the code layer that actually does the work, since it means fewer templates, thus potentially more code in cpp files, clearer compiler errors, clearer dependencies (e.g. the `Tags::ObservationLMax` needs to be added to the global cache somewhere, so hiding it in a function obstructs the dependency somewhat), etc. So suggest you just take `time`, `observation_l_max`, `l_max` and the `news` scalar as arguments here.